vendor/brightray/browser/inspectable_web_contents_impl.h

```diff
@@ -70,6 +69,8 @@ class InspectableWebContentsImpl :
   void UpdateDevToolsZoomLevel(double level);
 
   void WebContentsDestroyed() override;
+
+  void OnURLFetchComplete(const net::URLFetcher* source);
```

Comment:
Can we friend `FetchDelegate` or everyone can call this public method?

Reply:
originally I was going to do that, but it's in an anonymous namespace. This class is not going to be around much longer so I'm not too worried about it